docs/configuration.md

```diff
@@ -30,6 +30,8 @@
   are both on the same file (count, default 0 (no backpressure))
   e.g. backpressure = 10 [writer = 100.log, slowest reader = 89.log, delta = 11]
 * **hostname** - hostname used in logging/messages (default gethostname())
+* **remove_checkpoint_on_stop** - removes the checkpoint entry when the plugin
```

Comment:
I think a global option is ok.
My point was just that the "terminate" case was quite different than the "stop" case in the sense that stopping plugins is a normal process while plugins being terminated is unexpected (an issue occurs): so I thought it could make sense to have 2 different options to be able to handle the checkpoint differently for this 2 different situations.
But if you think it is better to keep a single option for both, it is also fine for me.